src/main/java/crazypants/enderio/machine/reservoir/BlockReservoir.java

```diff
@@ -59,7 +60,7 @@ public static BlockReservoir create() {
   }
 
   private BlockReservoir() {
-    super(MachineObject.blockReservoir.getUnlocalisedName(), TileReservoir.class, new Material(MapColor.WATER) {
+    super(MachineObject.blockReservoir, TileReservoir.class, new Material(MapColor.WATER) {
```

Comment:
the modObject is provided as parameter to the static create() and should not be hardcoded in the Block class